dev/Dockerfile

```diff
@@ -0,0 +1,66 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
where is this dockerfile being used?

Reply:
It's used by the `spark-iceberg` container in the docker-compose.yml file. It uses `build: .` to tell it to use this Dockerfile

Reply:
ah I missed that. In that case we should rename the image name, because I was assuming it's using `tabulario/spark-iceberg`. Also do we even need the image customization at this point?

Reply:
honestly i was just following the configuration that iceberg-python is using (the Dockerfile is from that repo) I'll see if it works without the customization and report back.